test/e2e/main_test.go

```diff
@@ -74,9 +75,15 @@ func TestE2E(tapi *testing.T) {
 	for _, node := range nodeList.Items {
 		nodes = append(nodes, node.Name)
 	}
-	knmstateReporter := knmstatereporter.New("test_logs/e2e", framework.Global.Namespace, nodes)
-	junitReporter := ginkgoreporters.NewJUnitReporter("junit.functest.xml")
-	RunSpecsWithDefaultAndCustomReporters(t, "E2E Test Suite", []Reporter{junitReporter, knmstateReporter})
+
+	reporters := make([]Reporter, 0)
+	reporters = append(reporters, knmstatereporter.New("test_logs/e2e", framework.Global.Namespace, nodes))
+	reporters = append(reporters, ginkgoreporters.NewJUnitReporter("junit.functest.xml"))
+	if qereporters.JunitOutput != "" {
+		reporters = append(reporters, qereporters.NewJunitReporter())
+	}
```

Comment:
Why this ?

Reply:
It is a special version of ginkgo reported maintained by QE. I guess it parses polarion IDs, integrates with BugZilla etc.